### ISSUE TITLE
False positive for Bundler/DuplicatedGem

### ISSUE THREAD

**Issue by marcandre** [rubocop-hq/rubocop#8918]
Should not complain

```ruby
  if File.exist?(local_ast)
    gem 'rubocop', path: local_ast
  elsif ENV['RUBOCOP_VERSION'] == 'master'
    gem 'rubocop', git: 'https://github.com/rubocop-hq/rubocop.git'
  else
    gem 'rubocop', '>= 1.0'
  end
```

cc/ @tejasbubane 

I didn't check, but `case/when` should also work...
